Documentation/AEPEdgeIdentity.md

```diff
@@ -0,0 +1,90 @@
+# Getting Started
+
+# Test App
```

Comment:
Should  mention this is about Android Test App  to be clear this is for Android?

Reply:
Added "Android" to header